.github/workflows/jira.yml

```diff
@@ -0,0 +1,34 @@
+name: Jira
+
+on:
+  pull_request:
+    branches: [main]
+    paths-ignore: ["*.md"]
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check Jira Story
+      shell: pwsh
+      run: |
+            Install-Module JiraPS -Scope CurrentUser -Force
+            if ("${{ github.event.pull_request.head.ref }}" -like "*/renovate/*" -eq $True)
```

Comment:
Semgrep seems to think doing this is no bueno and could lead to some form of attack. It suggest to go through an env var

Reply:
I'll try with [GITHUB_REF_NAME](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables)